packages/database/src/migrations/20210128055252-AddStrivePositiveResultBarGraph-modifies-data.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+import { insertObject } from '../utilities/migration';
+import { generateId } from '../utilities/generateId';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+
+const CODE_TO_NAME = { STR_PF05: 'PF', STR_PV05: 'PV', STR_PM05: 'PM', STR_PO05: 'PO' };
+
+const insertIndicator = (db, code) => {
+  const indicator = {
+    id: generateId(),
+    code: `STRIVE_PERCENTAGE_${CODE_TO_NAME[code]}_OF_TOTAL_WEEKLY`,
+    builder: 'arithmetic',
+    config: {
+      formula: `${code} / STR_QMAL05`,
+      aggregation: {
+        [code]: 'SUM_EACH_WEEK',
+        STR_QMAL05: 'SUM_EACH_WEEK',
+      },
+    },
+  };
+  insertObject(db, 'indicator', indicator);
+};
+
+const REPORT = {
+  id: 'PG_Strive_PNG_Positive_Result_By_Type_Bar',
+  dataBuilder: 'analyticsPerPeriod',
+  dataBuilderConfig: {
+    series: [
+      { key: 'Pf +ve', dataElementCode: 'STRIVE_PERCENTAGE_PF_OF_TOTAL_WEEKLY' },
+      { key: 'Pv +ve', dataElementCode: 'STRIVE_PERCENTAGE_PV_OF_TOTAL_WEEKLY' },
+      { key: 'Pm +ve', dataElementCode: 'STRIVE_PERCENTAGE_PM_OF_TOTAL_WEEKLY' },
+      { key: 'Po +ve', dataElementCode: 'STRIVE_PERCENTAGE_PO_OF_TOTAL_WEEKLY' },
+    ],
+    entityAggregation: {
```

Comment:
Note: because of the `SUM_EACH_WEEK` aggregationType for each dataElement in the indicators, there should only be one analytic per indicator per week returned to the dataBuilder.

This means that the entityAggregation here is mostly for show, the org unit should NOT be important in building the data. 

However, I figure it makes the most sense to still have it, as it matches conceptually what we are doing here.